app/services/notifications/setup/batch-notifications.service.js

```diff
@@ -6,11 +6,12 @@
  */
 
 const CreateNotificationsService = require('./create-notifications.service.js')
+const NotifyConfig = require('../../../../config/notify.config.js')
```

Comment:
As it's config it needs to go below our modules ([Top of .js files](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#top-of-js-files))